package-system/DirectXShaderCompiler/build_config.json

```diff
@@ -50,8 +52,10 @@
                "-DSPIRV_BUILD_TESTS:BOOL=ON"
             ],
             "cmake_build_args":[
+               "--target",
+               "all llvm-as llvm-dis",
                "-j",
-               "8"
+               "16"
```

Comment:
I'd recommend specifying 
```
--parallel
```

instead of using the -j parameter.  (and don't include a number)


instead.  This will pass the necessary command line parameters down to the builder and will work completely independently of OS or builder.  For example for gcc it will send -j(cores) and for ninja it will omit it since its the default, and so on

Reply:
so an example would be to replace
```
"-j",
"16"
```
with just
```
"--parallel"
```

Reply:
Thanks for the tip. changed to --parallel